docs/users/developing.md

```diff
@@ -388,6 +388,46 @@ $ open htmlcov/index.html
+
+:::{admonition} Information
+For interactive widgets to appear in rendered documentation, they must be stored in the notebook each time it is re-executed. For information on how to do this, see [](#widgets).
+:::
+
+[Jupyter Notebooks]: https://jupyter.org/
+[Jupyter widgets]: https://ipywidgets.readthedocs.io/
+[molecule cookbook]: molecule_cookbook.ipynb#from-pdb-file
+[OpenFF Examples]: https://docs.openforcefield.org/examples
+
+#### MyST Markdown
+
+Documentation notebooks are rendered with the [MyST-NB] Sphinx extension, which provides the expanded Markdown syntax used in other Markdown files in this documentation. Admonitions, maths, and links to API references and other documentation pages can be included in the usual ways. This means that some correct Markdown will appear incorrect in Jupyter by default, as it uses a simpler Markdown dialect. To render MyST syntax, you can install the [`jupyterlab-myst`] Jupyter extension:
+
+```shell
+mamba install -c conda-forge jupyterlab-myst
+```
+
+Note that some features that will render correctly in Sphinx rely on extensions that are not available to `jupyterlab-myst`, and so may still appear incorrect even with the extension.
+
+[MyST-NB]: https://myst-nb.readthedocs.io/
+[`jupyterlab-myst`]: https://mystmd.org/guide/quickstart-jupyter-lab-myst
+
+#### Widgets
+
+Jupyter Notebooks can produce interactive [widgets] as output, and MyST-NB supports rendering them in HTML output. Any given widget includes both Python code and JavaScript code, and Jupyter defines ways for them to speak to each other which allows Python code to be run interactively via the widget in the notebook. 
+
+While the JavaScript components are embedded in the page by Sphinx and can run in the browser's JavaScript engine, Sphinx web pages do not include a Python runtime. This has two effects. Firstly, any interactivity that relies on the Python runtime will not work on the web. This is usually little more than an inconvenience for purpose-written documentation notebooks, but does explain why [NGLView trajectories in OpenFF Examples] cannot be played on the web. 
+
+Secondly, widget initialization happens in Python, so any state shared between Python and JavaScript must be included in the notebook itself before Sphinx can render it. Any time these notebooks are re-executed, the internal IDs of the widgets may change, causing the widget to forget its original state and be missing from the rendered documentation. To fix this, it's essential to ask Jupyter to **save widget state**. In Jupyter Notebook 7+ and Jupyter Lab, this option appears under `Settings -> Save Widget State Automatically`, which if ticked will save widget state when the notebook is saved. Note that this menu item will not appear if the open notebook has no widgets. In earlier versions of Notebook, the `Widgets -> Save Widget State` menu item must be selected any time the notebook is re-executed and saved.
```

Comment:
Not blocking: is there a way to do this with `jupyter` CLI? Same question for VS code. Both questions are because I'm lazy

Reply:
CLI is addressed in the next paragraph as you noted, but as for Code - I have no idea, sorry!